package.json

```diff
@@ -53,7 +53,7 @@
     "postpublish": "yarn --cwd lib/cli test -o",
     "publish:alpha": "npm run publish -- --npm-tag=alpha",
     "repo-dirty-check": "node ./scripts/repo-dirty-check",
-    "start": "npm --prefix examples/official-storybook run storybook",
+    "start": "yarn --cwd examples/official-storybook storybook",
```

Comment:
does this require people to have yarn installed?

Reply:
not really, I believe this is only applicable to the storybook project itself

Reply:
> does this require people to have yarn installed?

You need it anyway when you contribute to storybook, see `CONTRIBUTING.md`